authorization/3.0/_index.md

```diff
@@ -25,42 +27,22 @@ The primary goal of this release is to make Jakarta Authorization future proof b
 
 * [Jakarta Authorization 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.authorization/releases/3.0)
 
-<!--
 The following can be uncommented and version information updated as they become available.
 
-* [Jakarta Authorization 3.0 Specification Document](./authorization-spec-3.0.pdf) (PDF)
-* [Jakarta Authorization 3.0 Specification Document](./authorization-spec-3.0.html) (HTML)
+* [Jakarta Authorization 3.0 Specification Document](./jakarta-authorization-spec-3.0.pdf) (PDF)
+* [Jakarta Authorization 3.0 Specification Document](./jakarta-authorization-spec-3.0.html) (HTML)
 * [Jakarta Authorization 3.0 Javadoc](./apidocs)
 * [Jakarta Authorization 3.0 TCK](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip)([sig](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/authorization/3.0/jakarta-authorization-tck-3.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
   * [jakarta.authorization:jakarta.authorization-api:jar:3.0.0](https://search.maven.org/artifact/jakarta.authorization/jakarta.authorization-api/3.0.0/jar)
--->
 
 # Compatible Implementations
 
-* Exousia (TBC)
 
-# Ballots
+* Exousia 3.0.0-M3 / GlassFish 8.0.0-M4
```

Comment:
Please mention [Eclipse Glassfish 8.0.0-JDK-17-M5](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/8.0.0-JDK17-M5/)

Reply:
It will be good to add the hyperlinks

Reply:
> Please mention [Eclipse Glassfish 8.0.0-JDK-17-M5](https://repo1.maven.org/maven2/org/glassfish/main/distributions/glassfish/8.0.0-JDK17-M5/)

It is done ;)